lib/doorkeeper/orm/active_record.rb

```diff
@@ -12,6 +12,27 @@ def self.initialize_models!
           end
         end
       end
+
+      def self.initialize_application_owner!
```

Comment:
If this method is the same in every ORM, should we leave the definition only in ` lib/doorkeeper/config.rb`?